contributors.txt

```diff
@@ -57,3 +57,4 @@ YYYY/MM/DD, github id, Full name, email
 2022/05/20, JoinedSenses, Arron Vinyard, jo**es@gmail.com
 2022/09/20, arlm, Alexandre Marcondes, ale**es@gmail.com
 2023/08/28, Ghost4Man, Daniel Kosík, kosik.dan@gmail.com
+2023/10/17, millergarym, miller.garym@gmail.com
```

Comment:
This is no longer needed, now that this repo uses DCO, but if you want you can leave it in there.